rule.go

```diff
@@ -71,31 +71,98 @@ type Rule struct {
 
 // GetRule returns the rules in the specified table and chain.
 //
-// Deprecated: use GetRules instead.
+// Deprecated: use GetRuleByHandle instead.
 func (cc *Conn) GetRule(t *Table, c *Chain) ([]*Rule, error) {
 	return cc.GetRules(t, c)
 }
 
+// GetRuleByHandle returns the rule in the specified table and chain by its
+// handle.
+// https://docs.kernel.org/networking/netlink_spec/nftables.html#getrule
+func (cc *Conn) GetRuleByHandle(t *Table, c *Chain, handle uint64) (*Rule, error) {
```

Comment:
Here I assumed that we want to keep consistency with the rest of the rule methods in the file. It could have been `(c *Chain, handle uint64)` instead but I don't it'd be clear to the user that the Table pointer cannot be nil.

Reply:
> It could have been `(c *Chain, handle uint64)` instead but I don't think it'd be clear to the user that the Table pointer cannot be nil.

I’m having trouble parsing this sentence. The code looks like `*Table` is required, and you say `the Table pointer cannot be nil`, so then how could this signature possibly be `(c *Chain, handle uint64)`  (no Table pointer)…?

Reply:
Sorry I meant the *Table pointer inside of the chain object. I am not sure whether users would expect &Chain{Name: "test-chain} to be an invalid input.